sage-ios/SAGE/SAGE/AnnouncementsViewController.swift

```diff
@@ -218,33 +240,56 @@ class AnnouncementsViewController: SGTableViewController {
     // MARK: - UITableViewDelegate
     //
     override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
-        return AnnouncementsTableViewCell.heightForAnnouncement(announcements[indexPath.row], width: CGRectGetWidth(tableView.frame))
+        if indexPath.section == 1 {
+            return SGLoadingCell.cellHeight
+        } else {
+            return AnnouncementsTableViewCell.heightForAnnouncement(self.announcements[indexPath.row], width: CGRectGetWidth(tableView.frame))
+            
+        }
     }
     
     override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
-        return 1
+        return 2
     }
     
     override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return announcements.count
+        if section == 1 {
+            if self.loadedAllAnnouncements {
+                return 0
+            } else {
+                return 1
+            }
+        } else {
+            return self.announcements.count
+        }
     }
     
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        var cell = tableView.dequeueReusableCellWithIdentifier("Announcement")
-        if (cell == nil) {
-            cell = AnnouncementsTableViewCell(style:UITableViewCellStyle.Default, reuseIdentifier:"Announcement")
+        if indexPath.section == 1 {
+            let loadingCell = SGLoadingCell()
+            loadingCell.userInteractionEnabled = false
```

Comment:
it has this property as true by default because thats the tableviewproperty i believe